components/gitpod-db/src/typeorm/transformer.ts

```diff
@@ -38,6 +38,18 @@ export namespace Transformer {
         },
     };
 
+    export const MAP_ISO_STRING_TO_TIMESTAMP_DROP_NULLABLE: ValueTransformer = {
```

Comment:
Interesting. I know that we store times as both ISO strings and timestamps in MySQL. I'm just curious -- what is the benefit of converting ISO string to timestamp here? Is it more efficient to store timestamps in MySQL? 👀

Reply:
The main reason is that other DBs in the repo are set up like this🤣